agent/src/runtime_connectors/podman/podman_runtime.rs

```diff
@@ -169,7 +165,7 @@ impl RuntimeConnector<PodmanWorkloadId, GenericPollingStateChecker> for PodmanRu
 
     // [impl->swdd~podman-delete-workload-stops-and-removes-workload~1]
     async fn delete_workload(&self, workload_id: &PodmanWorkloadId) -> Result<(), RuntimeError> {
-        log::info!("Deleting workload with id '{}'", workload_id.id);
+        log::debug!("Deleting workload with id '{}'", workload_id.id);
```

Comment:
Short question, sometimes I think it is helpful without turning the log to debug that important steps are "documented" for the user in the logs, that the user can see that at least the agent tries to do something ... "deleting the workload". Or what do you think?

```suggestion
        log::info!("Deleting workload with id '{}'", workload_id.id);
```